cityelf/src/main/java/com/cityelf/repository/IRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.cityelf.repository;
+
+import com.cityelf.model.Notification;
+import com.cityelf.model.User;
+
+/**
+ * Created by Sergii on 26.04.2017.
```

Comment:
This comment is redundant. Git saves all the information about who changed the file and when.
Disable automatic insertion of that thing in idea.